pinot-core/src/main/java/com/linkedin/pinot/core/segment/store/SegmentLocalFSDirectory.java

```diff
@@ -75,7 +75,34 @@ public Path getPath() {
 
   @Override
   public long getDiskSizeBytes() {
-    return FileUtils.sizeOfDirectory(segmentDirectory.toPath().toFile());
+    // [PINOT-3479] For newly added refresh segments, the new segment will
```

Comment:
Jira ticket number is fine (at least until we get an external JIRA, if that ever happens), though it would help if besides the ticket number it gives a short summary of the ticket (eg. PINOT-1234 (NPE under certain conditions when ...))